iceberg/iceberg-handler/src/test/results/positive/llap/vectorized_iceberg_read_mixed.q.out

```diff
@@ -48,21 +48,25 @@ STAGE PLANS:
             Map Operator Tree:
                   TableScan Vectorization:
                       native: true
-                      vectorizationSchemaColumns: [0:a:int, 1:b:string]
-                    Group By Vectorization:
-                        aggregators: VectorUDAFMaxLong(col 0:int) -> int
-                        className: VectorGroupByOperator
-                        groupByMode: HASH
-                        keyExpressions: col 1:string
-                        native: false
-                        vectorProcessingMode: HASH
-                        projectedOutputColumnNums: [0]
-                      Reduce Sink Vectorization:
-                          className: VectorReduceSinkStringOperator
-                          keyColumns: 0:string
-                          native: true
-                          nativeConditionsMet: hive.vectorized.execution.reducesink.new.enabled IS true, hive.execution.engine tez IN [tez] IS true, No PTF TopN IS true, No DISTINCT columns IS true, BinarySortableSerDe for keys IS true, LazyBinarySerDe for values IS true
-                          valueColumns: 1:int
+                      vectorizationSchemaColumns: [0:a:int, 1:b:string, 2:PARTITION__SPEC__ID:int, 3:PARTITION__HASH:bigint, 4:FILE__PATH:string, 5:ROW__POSITION:bigint]
```

Comment:
why did we get virtual columns here?

Reply:
The qtest use iceberg v1 table, but in this PR we use iceberg v2 table by default, and so the execution plan is change. That is to say, This difference has nothing to do with the PR.

https://github.com/apache/hive/pull/3324/files#diff-74e083aeb557dfd57f4fa0b93b1617843ac8d4e61f1aff9ec91df1945cebb5a3
I think this change is related HIVE-26264, but i am not sure if it is a normal change. @kasakrisz Could you please help to check why the execution plan is different between v1 and v2 table? Thanks in advance.

Reply:
Virtual columns are added to the TS schema if the table is an ACID table:
https://github.com/apache/hive/blob/a4aef8173269d213eb13666baf1751bf110579c8/ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java#L11946C32-L11946C74
An Iceberg V2 table is ACID but a V1 is not:
https://github.com/apache/hive/blob/a4aef8173269d213eb13666baf1751bf110579c8/iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java#L1063-L1074

Reply:
Got it! Thanks for your explanation. That is to say, this is normal change. CC @deniskuzZ

Reply:
won't we loose then test cases for v1 tables? shouldn't we update the q file and explicitly set v1?

Reply:
Ok, Changed some tests back to v1 to keep its original execution plan. https://github.com/apache/hive/pull/4824/commits/e363797e1f8af80dd55ef2388408d41fc2ddcfc7
`vectorized_iceberg_read_orc.q,
vectorized_iceberg_read_mixed.q,
vectorized_iceberg_read_parquet.q`

Reply:
👍